test/map-spec.js

```diff
@@ -1,34 +1,39 @@
 import MapGL from '../src';
 
-import document from 'global/document';
-/* global process */
 import React from 'react';
 import ReactDOM from 'react-dom';
+import document from 'global/document';
 import test from 'tape-catch';
 
-/* eslint-disable no-process-env */
```

Comment:
I lost some context here, did we decide to go with `// esline-disable-line` instead of the multi-line `/* eslint-disable .... */`?